lib/loader.js

```diff
@@ -9,16 +9,33 @@ const postCssPlugins = require('./postcss-plugins');
 const extractThemeRulesPlugin = postCssPlugins.extractThemeRulesPlugin;
 const removeCssModuleExports = postCssPlugins.removeCssModuleExports;
 
-// copied from css-loader/lib/getLocalIdent.js
+// copied from css-loader/src/utils.js
```

Comment:
We could maybe import directly from `css-loader` like
```js
const { getLocalIdent } = require('css-loader/src/utils')
```
What we have here is fine too.